cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml

```diff
@@ -41,10 +41,8 @@ spec:
               cpu: 100m
               memory: 100Mi
           env: 
-            - name: INFLUXDB_EXTERNAL_URL
-              value: /api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:api/db/
```

Comment:
The version of Grafana that is currently used by K8s uses client side js to
fetch data from influxdb. Unless Influxdb is exposed as a public service,
how will Grafana access the DB?
In any case, heapster head has been updated to use Grafana 2.0. I plan on
updating k8s to use that version.

On Mon, Sep 14, 2015 at 5:12 PM, Pedro Marques <notifications@github.com>
wrote:

> In
> cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-controller.yaml
> <https://github.com/kubernetes/kubernetes/pull/13942#discussion_r39462624>
> :
>
> > @@ -41,10 +41,8 @@ spec:
> >                cpu: 100m
> >                memory: 100Mi
> >            env:
> > -            - name: INFLUXDB_EXTERNAL_URL
> > -              value: /api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb:api/db/
>
> @vishh <https://github.com/vishh> Not sure if i follow. This is an env
> var that will only affect this pod, correct ?
> The api/v1/proxy URL is still available on the apiserver (as per kubectl
> cluster-info).
> The current configuration (pre PR) is asking grafana to go through
> kube-proxy which will map the connection back to the same pod, correct ? It
> actually doesn't send the traffic to the api-server, right ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13942/files#r39462624>.
>